ml_models/bpe_model.py

```diff
@@ -0,0 +1,71 @@
+from functools import wraps
+import numpy as np
+from text_utils.indexer import *
+from text_utils.utils import create_logger
+from config import logger_path
+from bpemb import BPEmb
+
+logger = create_logger(__name__, logger_path['bpe'])
+
+FEATURE_SIZE = 300
+
+def singleton(cls):
+    instance = None
+
+    @wraps(cls)
+    def inner(*args, **kwargs):
+        nonlocal instance
+        if instance is None:
+            instance = cls(*args, **kwargs)
+        return instance
+
+    return inner
+
+class BPEEmbedder:
```

Comment:
Possible we should make a base interface to these models, and then implement it
```
import abc
class BasicEmbedder(abc.ABC):
    def sentence_embedding(text):
        pass
```

Reply:
To be implemented in separate patch.